.eleventy.js

```diff
@@ -222,6 +222,21 @@ module.exports = function (eleventyConfig) {
       </ul>`;
   });
 
+  eleventyConfig.addPairedShortcode('componentPreview', (children, title, padding = "py-400", margin = "my-500") => {
+    const content = markdownLibrary.render(children);
```

Comment:
Since we are only passing HTML and our web components to this paired shortcode, you don't need to run this through the Markdown render and can just pass `children` below.